modules/openapi-generator/src/main/resources/typescript-inversify/api.service.mustache

```diff
@@ -8,7 +8,7 @@ import { Observable } from "rxjs/Observable";
 import { Observable } from "rxjs";
 {{/useRxJS6}}
 
-import { map } from "rxjs/operators";
+import { GlobalImportOperators } from "rxjs/operators";
```

Comment:
`"rxjs/operators"` does not export a `GlobalImportOperators` symbol

Reply:
Ah I got it....

Reply:
so we need something like
```
import * as GlobalImportOperators from "rxjs/operators";
```

and then call it by 
```
GlobalImportOperators.map
```
(or something like that) ?